ALGORITHMS.md

```diff
@@ -228,58 +232,58 @@ If [OQS_KEM_ENCODERS](CONFIGURE.md#OQS_KEM_ENCODERS) is enabled the following li
-| x448_frodo976aes | 1.3.9999.99.47 | OQS_OID_X448_FRODO976AES
-| frodo976shake | 1.3.9999.99.67 | OQS_OID_FRODO976SHAKE
-| p384_frodo976shake | 1.3.9999.99.66 | OQS_OID_P384_FRODO976SHAKE
-| x448_frodo976shake | 1.3.9999.99.48 | OQS_OID_X448_FRODO976SHAKE
-| frodo1344aes | 1.3.9999.99.69 | OQS_OID_FRODO1344AES
-| p521_frodo1344aes | 1.3.9999.99.68 | OQS_OID_P521_FRODO1344AES
-| frodo1344shake | 1.3.9999.99.71 | OQS_OID_FRODO1344SHAKE
-| p521_frodo1344shake | 1.3.9999.99.70 | OQS_OID_P521_FRODO1344SHAKE
+| frodo640aes | NULL | OQS_OID_FRODO640AES
+| p256_frodo640aes | NULL | OQS_OID_P256_FRODO640AES
+| x25519_frodo640aes | NULL | OQS_OID_X25519_FRODO640AES
+| frodo640shake | NULL | OQS_OID_FRODO640SHAKE
+| p256_frodo640shake | NULL | OQS_OID_P256_FRODO640SHAKE
+| x25519_frodo640shake | NULL | OQS_OID_X25519_FRODO640SHAKE
+| frodo976aes | NULL | OQS_OID_FRODO976AES
+| p384_frodo976aes | NULL | OQS_OID_P384_FRODO976AES
+| x448_frodo976aes | NULL | OQS_OID_X448_FRODO976AES
+| frodo976shake | NULL | OQS_OID_FRODO976SHAKE
+| p384_frodo976shake | NULL | OQS_OID_P384_FRODO976SHAKE
+| x448_frodo976shake | NULL | OQS_OID_X448_FRODO976SHAKE
+| frodo1344aes | NULL | OQS_OID_FRODO1344AES
+| p521_frodo1344aes | NULL | OQS_OID_P521_FRODO1344AES
+| frodo1344shake | NULL | OQS_OID_FRODO1344SHAKE
+| p521_frodo1344shake | NULL | OQS_OID_P521_FRODO1344SHAKE
 | kyber512 | 1.3.6.1.4.1.2.267.8.2.2 | OQS_OID_KYBER512
-| p256_kyber512 | 1.3.9999.99.72 | OQS_OID_P256_KYBER512
-| x25519_kyber512 | 1.3.9999.99.49 | OQS_OID_X25519_KYBER512
+| p256_kyber512 | NULL | OQS_OID_P256_KYBER512
+| x25519_kyber512 | NULL | OQS_OID_X25519_KYBER512
 | kyber768 | 1.3.6.1.4.1.2.267.8.3.3 | OQS_OID_KYBER768
-| p384_kyber768 | 1.3.9999.99.73 | OQS_OID_P384_KYBER768
-| x448_kyber768 | 1.3.9999.99.50 | OQS_OID_X448_KYBER768
-| x25519_kyber768 | 1.3.9999.99.51 | OQS_OID_X25519_KYBER768
-| p256_kyber768 | 1.3.9999.99.52 | OQS_OID_P256_KYBER768
+| p384_kyber768 | NULL | OQS_OID_P384_KYBER768
+| x448_kyber768 | NULL | OQS_OID_X448_KYBER768
+| x25519_kyber768 | NULL | OQS_OID_X25519_KYBER768
+| p256_kyber768 | NULL | OQS_OID_P256_KYBER768
 | kyber1024 | 1.3.6.1.4.1.2.267.8.4.4 | OQS_OID_KYBER1024
-| p521_kyber1024 | 1.3.9999.99.74 | OQS_OID_P521_KYBER1024
+| p521_kyber1024 | NULL | OQS_OID_P521_KYBER1024
 | mlkem512 | 2.16.840.1.101.3.4.4.1 | OQS_OID_MLKEM512
 | p256_mlkem512 | 1.3.6.1.4.1.22554.5.7.1 | OQS_OID_P256_MLKEM512
 | x25519_mlkem512 | 1.3.6.1.4.1.22554.5.8.1 | OQS_OID_X25519_MLKEM512
 | mlkem768 | 2.16.840.1.101.3.4.4.2 | OQS_OID_MLKEM768
-| p384_mlkem768 | 1.3.9999.99.75 | OQS_OID_P384_MLKEM768
-| x448_mlkem768 | 1.3.9999.99.53 | OQS_OID_X448_MLKEM768
-| x25519_mlkem768 | 1.3.9999.99.54 | OQS_OID_X25519_MLKEM768
-| p256_mlkem768 | 1.3.9999.99.55 | OQS_OID_P256_MLKEM768
+| p384_mlkem768 | NULL | OQS_OID_P384_MLKEM768
+| x448_mlkem768 | NULL | OQS_OID_X448_MLKEM768
+| x25519_mlkem768 | NULL | OQS_OID_X25519_MLKEM768
+| p256_mlkem768 | NULL | OQS_OID_P256_MLKEM768
```

Comment:
You are dropping the two standardized `*_mlkem768` hybrids while retaining the unused `*_mlkem512` ones, shouldn't it be the other way around?